controllers/operands/kubevirt.go

```diff
@@ -736,6 +768,21 @@ func getKVDevConfig(hc *hcov1beta1.HyperConverged) *kubevirtcorev1.DeveloperConf
 	return devConf
 }
 
+func primaryUserDefinedNetworkBinding(sidecarImage string) kubevirtcorev1.InterfaceBindingPlugin {
+	return kubevirtcorev1.InterfaceBindingPlugin{
+		NetworkAttachmentDefinition: path.Join(primaryUDNNetworkBindingNamespace, primaryUDNNetworkBindingName),
+		SidecarImage:                sidecarImage,
+		Migration: &kubevirtcorev1.InterfaceBindingMigration{
+			Method: kubevirtcorev1.LinkRefresh,
+		},
+		ComputeResourceOverhead: &corev1.ResourceRequirements{
+			Requests: corev1.ResourceList{
+				corev1.ResourceMemory: resource.MustParse("500Mi"),
```

Comment:
This magic number of 500Mi is not clear. Can we use a constant with meaningful name, that will self explain it?

Reply:
DONE.